cadence/templates/server-job.yaml

```diff
@@ -110,7 +110,14 @@ spec:
             - name: SQL_USER
               value: {{ include "cadence.persistence.sql.user" (list $ $store) }}
             - name: SQL_PASSWORD
+              {{- if or .Values.cassandra.enabled .Values.mysql.enabled }}
```

Comment:
**Question**: is this condition the right one to use here?

We are in the `{{- if eq (include "cadence.persistence.driver" (list $ $store)) "sql" }}` condition from line 101 so the persistence driver is known to be `sql`, in which case I believe we should only care about whether `mysql` is enabled because for Cassandra Cadence use case the driver would be `cassandra` as well and the `SQL_PASSWORD` environment variable wouldn't be used.

If there is no reason to include the Cassandra enabled check then I propose removing it from the condition.

```suggestion
              {{- if .Values.mysql.enabled }}
```

The logic is also similar at lines 168 and 299 if we decide to modify it.
I'm going to add references to those lines as well for tracking and web-UI suggestion committing purposes.

Reply:
I will scope this down. Good catch!